Error: use of undeclared identifier 'equationIndexes' in generated code

Please consider the attached test case [GridBaseBreakers.mo.txt](https://github.com/OpenModelica/OpenModelica/files/8096743/GridBaseBreakers.mo.txt).

The generated C code is invalid and produces a number of errors during compilation, e.g.
```
GridBaseBreakers_05evt.c:122:57: error: use of undeclared identifier 'equationIndexes'
      infoStreamPrintWithEquationIndexes(LOG_ASSERT, 0, equationIndexes, "The following assertion has been violated %sat time %f", initial() ? "during initialization " : "", data->localData[0]->timeValue);
```

```
GridBaseBreakers_05evt.c:127:55: error: use of undeclared identifier 'equationIndexes'
      throwStreamPrintWithEquationIndexes(threadData, equationIndexes, "Model error: Argument of sqrt(i_n[1,1].re ^ 2.0 + i_n[1,1].im ^ 2.0) was %g should be >= 0", tmp2);
```
@mahge, could you give it a look?

Keeping @bernhardbachmann mann in the loop.

--- a/OMCompiler/Compiler/Template/CodegenC.tpl
+++ b/OMCompiler/Compiler/Template/CodegenC.tpl
@@ -4655,6 +4655,8 @@ template functionZeroCrossing(list<ZeroCrossing> zeroCrossings, list<SimEqSystem
   int <%symbolName(modelNamePrefix,"function_ZeroCrossings")%>(DATA *data, threadData_t *threadData, double *gout)
   {
     TRACE_PUSH
+    const int *equationIndexes = NULL;
+
     <%varDecls2%>
 
   #if !defined(OMC_MINIMAL_RUNTIME)
@@ -4796,6 +4798,8 @@ template functionRelations(list<ZeroCrossing> relations, String modelNamePrefix)
   int <%symbolName(modelNamePrefix,"function_updateRelations")%>(DATA *data, threadData_t *threadData, int evalforZeroCross)
   {
     TRACE_PUSH
+    const int *equationIndexes = NULL;
+
     <%varDecls%>
 
     if(evalforZeroCross) {

